test/gem_test.rb

```diff
@@ -9,11 +9,11 @@ class GemTest < MiniTest::Spec
 
     # We're going to muck with some system globals so lets make sure
     # they get set back later
-    original_host = RbConfig::CONFIG['host']
+    original_platform = RbConfig::CONFIG["arch"]
```

Comment:
Prefer single quote string literals to match elsewhere in the file

Reply:
addressed in [cb98ec1](https://github.com/rails-sqlserver/tiny_tds/pull/533/commits/cb98ec137aa9b7dd20a8fcff78890b40ea682a48).